### PR TITLE
An image remains selected when switching between folders

### LINKED ISSUE #1117: An image remains selected when switching between folders
This behavior might be confusing for the user.

### Preconditions (*)
Magento 2.4-develop
Adobe-Stock-Integration 1.1-develop

1. Installed Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Enable Enhanced Media Gallery in Stores -> Configuration -> Advanced -> System 
enhansedMG
4. An image added to Media Gallery

### Steps to reproduce (*)
1. From Admin go to **Content** - **Pages**, click **Add New Page**
2. Expand **Content**,click **Show / Hide Editor**, click **Insert Image...**
3. Select a folder from the folder tree
4. Select an image from the selected folder 
_Add Selected_ button became active
5. Try to: 
- Switch between folders
- Click Search Adobe Stock Button
- Click away from the selected image
- Click Cancel button 

### Expected result (*)
The image is deselected

### Actual result (*)
The image is remains selected and the Add Selected button is active.
im_selection

### PR DESCRIPTION
**Pull request by engcom-Charlie**
### Description (*)

Closes #1117 

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1117: An image remains selected when switching between folders
2. ...

### Manual testing scenarios (*)

1. ...
2. ...

### CODE REVIEW

File: MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ define([
             deleteSelectedBtnSelector: '#delete_selected',
             targetElementId: null,
             selected: null,
-            mediaGalleryDirectoryComponent: 'media_gallery_listing.media_gallery_listing.media_gallery_directories',
+            mediaGalleryDirectoryComponent: '${ $.ns }.${ $.ns }.media_gallery_directories',
```
Comment (Nazar65) [inline MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js]: It would be better to move name space to xml configuration

**Review by Nazar65 — CHANGES_REQUESTED** [review]

File: MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js (hunk 1 above)
Comment (Nazar65) [reply MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js]: https://github.com/magento/adobe-stock-integration/blob/36b8d15c61f168969aa39a1e8539cd20a24cf902/MediaGalleryUi/view/adminhtml/ui_component/media_gallery_listing.xml#L178

**Comment by engcom-Charlie** [comment]
@magento run Unit tests

**Comment by engcom-Charlie** [comment]
@magento run Functional Tests CE

**Review by sivaschenko — APPROVED** [review]

**Comment by lenaorobei** [comment]
Please address unit tests failures

**Comment by chalov-anton** [comment]
:heavy_check_mark: QA Passed

The image is deselected after switching to another folder 

image deselected_2020-04-22 14-44

**Pull request merged**